tests/integration/scenarios/entities.py

```diff
@@ -17,7 +17,7 @@ class Pet(SQLModelEntity, table=True):
     """Pet model"""
 
     id: int = Field(index=True, default=None, primary_key=True)
```

Comment:
can't we get rid of this because we get this already from the `SQLModelEntity` parent class?